pkg/cmd/codespace/list.go

```diff
@@ -43,6 +43,13 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 		return fmt.Errorf("error getting codespaces: %w", err)
 	}
 
+	hasNonProdVSCSTarget := false
+	for _, apiCodespace := range codespaces {
+		if apiCodespace.VSCSTarget != "prod" {
```

Comment:
I think the production name is `production` not `prod`. Also I believe the field will not be in the API response at all unless you are a `codespaces_developer`, so we should be sure to handle a nil field the same as `production`.

Reply:
Updated name and support empty